mcs/class/System.Drawing/System.Drawing/BufferedGraphicsContext.cs

```diff
@@ -52,10 +52,9 @@ public BufferedGraphics Allocate (Graphics targetGraphics, Rectangle targetRecta
 			return graphics;			
 		}
 
-		[MonoTODO ("The targetDC parameter has no equivalent in libgdiplus.")]
 		public BufferedGraphics Allocate (IntPtr targetDC, Rectangle targetRectangle)
 		{
-			throw new NotImplementedException ();
+            return Allocate(Graphics.FromHdc(targetDC), targetRectangle);
```

Comment:
According to the http://www.mono-project.com/community/contributing/coding-guidelines/ please use tabs instead of spaces.